tests/filter.at.R

```diff
@@ -0,0 +1,132 @@
+library(data.table)
```

Comment:
Could you add `options(datatable.auto.index=FALSE)` in both of your test scripts? Just to ensure fair timing of all queries.